### ISSUE TITLE
Test for WCAG failure F70: incorrect use of start and end tags or attribute markup

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#161]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/F70

The objective of this failure is to identify examples of markup errors in element tags that could cause assistive technology to be unable to generate a satisfactory model of the page. Different user agents may implement different heuristics to recover from errors, resulting in inconsistent presentations of the page between user agents.

Some common types of problems with start and end tags that lead to this failure condition (though this is not an exhaustive list):
- [x] Opening and closing tags that are missing the opening < and closing > brackets.
- [x] Closing tag missing the initial / to indicate it is a closing tag.
- [x] Attribute values that have an opening quote but not a closing quote. Attribute values must be either fully quoted or, in some markup languages, may be unquoted.
- [x] Lack of whitespace between attributes.
- [x] Unquoted attribute values that have whitespace in the value.
- [x] Failure to provide a closing element tag for elements that do not accept empty-element syntax.
## Test procedure
1. Check the source code of pages implemented in markup languages.
2. Check whether any opening tags, closing tags or attributes are malformed.
